### ISSUE TITLE
Training of tless-refiner

### ISSUE THREAD

**Issue by azad96** [ylabbe/cosypose#32]
I started the training of the refiner with the following command
`runjob --ngpus=4 --queue=local python -m cosypose.scripts.run_pose_training --config tless-refiner`

However, the process is stuck at some point as it can be seen from the below screenshot.
![Screenshot from 2021-01-04 12-54-58](https://user-images.githubusercontent.com/39907647/103522891-1636a500-4e8c-11eb-9c0b-095cc4046547.png)

What can be the reason that the training does not end?

**Comment by ylabbe** [comment]
Hi, I have done limited experiments with multi-gpu training on a single machine. I would say there is a high chance it stops because one of the processes has crashed for some reason. Could you look at the outputs of the other processes ?

**Comment by azad96** [comment]
@ylabbe, I started a training with debug mode with the following command
`runjob --ngpus=4 --queue=local python -m cosypose.scripts.run_pose_training --config tless-coarse --debug`
Then, these are the processes that were created.
![Screenshot from 2021-01-05 10-05-58](https://user-images.githubusercontent.com/39907647/103616832-33c44700-4f3e-11eb-9158-1c1fddfa6d2a.png)
I checked the output of each one with
`tail -f /proc/<pid>/fd/1`
but all are the same which is below.
![Screenshot from 2021-01-05 10-06-10](https://user-images.githubusercontent.com/39907647/103616938-5fdfc800-4f3e-11eb-8800-bb12a10ee70d.png)
By the way, everything seems okay when I use 1 gpu.
